spec.html

```diff
@@ -10440,6 +10440,25 @@ <h1>Static Semantics: CodePointAt ( _string_, _position_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-stringindexof">
+      <h1>Static Semantics: StringIndexOf ( _string_, _searchValue_, _fromIndex_ )</h1>
+      <p>The abstract operation _StringIndexOf_ performs the following steps:</p>
+      <!-- Note: OOB fromIndex values (fromIndex >= string.length) are accepted by the algorithm. -->
```

Comment:
I've kept these comments for now since they seem useful, but let me know if you'd rather have them removed.

Reply:
would this be useful as a non-normative note, rather than a comment?

Reply:
Done.